packages/assets/src/loader/parsers/LoaderParser.ts

```diff
@@ -42,6 +42,9 @@ export interface LoaderParser<ASSET = any, META_DATA = any, CONFIG = Record<stri
     /** A config to adjust the parser */
     config?: CONFIG;
 
+    /** The name of the parser (this can be used when specifying loadParser in a LoadAsset) */
+    name: string;
```

Comment:
This should probably be optional, right? `_validateParsers` handles it as optional.